README.md

```diff
@@ -9,6 +9,10 @@ JavaScript with React Native.
 
 We target iOS initially, with Android version following shortly.
 
+**Note**: For GSoC, we will be focused on making RN app better rather
+than maintaining [Java Android app](https://github.com/zulip/zulip-android)
+and RN app in parallel. For more clarification, please go through [our detailed plan](https://github.com/zulip/zulip-android/blob/master/android-strategy.md).
```

Comment:
I think this should be more focused on how we plan for this to be the default Zulip app on Android as well.